Fix Regression Introduced In #20

nikeee opened the pull request:
`npm start`/webpack bundling fails because the presets `es2015`/`stage-1` were changed to `env` in #20.

Error log:
```
Version: webpack 2.4.1
Time: 515ms
                 Asset       Size  Chunks             Chunk Names
             bundle.js    4.36 kB       0  [emitted]  main
         bundle.js.map    2.62 kB       0  [emitted]  main
       assets/main.css  688 bytes          [emitted]  
        assets/dog.png     167 kB          [emitted]  
locales/de/common.json   51 bytes          [emitted]  
locales/en/common.json   45 bytes          [emitted]  
            index.html  783 bytes          [emitted]  
   [0] ./src/index.js 1.57 kB {0} [built] [failed] [1 error]
   [1] multi ./src/index.js 28 bytes {0} [built]

ERROR in ./src/index.js
Module build failed: Error: Couldn't find preset "es2015" relative to directory "/tmp/webengineering-2017/src/main/frontend/src"
    at /tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:293:19
    at Array.map (native)
    at OptionManager.resolvePresets (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
    at OptionManager.mergePresets (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
    at OptionManager.mergeOptions (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
    at OptionManager.init (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-loader/lib/index.js:48:20)
    at Object.module.exports (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-loader/lib/index.js:163:20)
 @ multi ./src/index.js
```

nikeee committed:
Fix Regression Introduced In #20

`npm start`/webpack bunding fails because `es2015`/`stage-1` was changed to `env` in #20.

mischah commented:
Hej @nikeee ,

It seems like it’s missing the preset because you didn’t install it.
Please double check that you made an `npm install` after pulling the changes.

To get rid of unused dependencies you actually should trash your node modules before installing:

```
rm -rf node_modules
npm i
```

Cheers, Michael

nikeee commented:
Here is my full reproduction:
```Shell
user@host /tmp: git clone git@github.com:nikeee/webengineering-2017.git
user@host /tmp:  cd webengineering-2017
user@host /tmp/webengineering-2017: git checkout e911ac77cda0e1f518e459ea132d0d7fa763000d # merge of #20
user@host /tmp/webengineering-2017: ➦ e911ac7: cd src/main/frontend 
user@host /tmp/webengineering-2017/src/main/frontend: ➦ e911ac7: npm i
npm notice created a lockfile as package-lock.json. You should commit this file.
added 497 packages in 13.787s
user@host /tmp/webengineering-2017/src/main/frontend: ➦ e911ac7: npm start

> webengineering-SS2017@1.0.0 start /tmp/webengineering-2017/src/main/frontend
> node ./node_modules/.bin/webpack -w


Webpack is watching the files…

Hash: d92aa19193d2db1c7d84
Version: webpack 2.4.1
Time: 503ms
                 Asset       Size  Chunks             Chunk Names
             bundle.js    4.36 kB       0  [emitted]  main
         bundle.js.map    2.62 kB       0  [emitted]  main
        assets/dog.png     167 kB          [emitted]  
       assets/main.css  688 bytes          [emitted]  
locales/de/common.json   51 bytes          [emitted]  
locales/en/common.json   45 bytes          [emitted]  
            index.html  783 bytes          [emitted]  
   [0] ./src/index.js 1.57 kB {0} [built] [failed] [1 error]
   [1] multi ./src/index.js 28 bytes {0} [built]

ERROR in ./src/index.js
Module build failed: Error: Couldn't find preset "es2015" relative to directory "/tmp/webengineering-2017/src/main/frontend/src"
    at /tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:293:19
    at Array.map (native)
    at OptionManager.resolvePresets (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
    at OptionManager.mergePresets (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
    at OptionManager.mergeOptions (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
    at OptionManager.init (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-loader/lib/index.js:48:20)
    at Object.module.exports (/tmp/webengineering-2017/src/main/frontend/node_modules/babel-loader/lib/index.js:163:20)
 @ multi ./src/index.js
^C%                                                                                                                                                                 

user@host /tmp/webengineering-2017/src/main/frontend: ➦ e911ac7: git checkout a3ac0896469b0aa60e788e0d8ee9dc118948fdc2 # this PR
Previous HEAD position was e911ac7... Merge branch 'master' into replace-babel-presets
HEAD is now at a3ac089... Fix Regression Introduced In #20
user@host /tmp/webengineering-2017/src/main/frontend: ➦ a3ac089: rm -rf node_modules 
user@host /tmp/webengineering-2017/src/main/frontend: ➦ a3ac089: rm package-lock.json 
user@host /tmp/webengineering-2017/src/main/frontend: ➦ a3ac089: npm i
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN react-dom@15.6.1 requires a peer of react@^15.6.1 but none was installed.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 497 packages in 10.673s
user@host /tmp/webengineering-2017/src/main/frontend: ➦ a3ac089: npm start

> webengineering-SS2017@1.0.0 start /tmp/webengineering-2017/src/main/frontend
> webpack -w


Webpack is watching the files…

Hash: b70cfec27bbb650bd494
Version: webpack 2.4.1
Time: 6866ms
                 Asset       Size  Chunks                    Chunk Names
             bundle.js    1.88 MB       0  [emitted]  [big]  main
         bundle.js.map    1.86 MB       0  [emitted]         main
        assets/dog.png     167 kB          [emitted]         
       assets/main.css  688 bytes          [emitted]         
locales/de/common.json   51 bytes          [emitted]         
locales/en/common.json   45 bytes          [emitted]         
            index.html  783 bytes          [emitted]         
   [0] ./~/react/react.js 56 bytes {0} [built]
  [21] ./~/react-dom/index.js 59 bytes {0} [built]
  [56] ./src/util/User.js 2.32 kB {0} [built]
 [199] ./~/react-cookie/lib/index.js 788 bytes {0} [built]
 [222] ./~/react-i18next/dist/commonjs/index.js 1.04 kB {0} [built]
 [223] ./~/react-router-dom/es/Link.js 3.82 kB {0} [built]
 [224] ./~/react-router-dom/es/index.js 925 bytes {0} [built]
 [231] ./src/index.js 5.61 kB {0} [built]
 [254] ./src/i18n.js 834 bytes {0} [built]
 [505] ./~/react-i18next/dist/commonjs/trans.js 6.98 kB {0} [built]
 [514] ./~/react-router-dom/es/Router.js 49 bytes {0} [built]
 [515] ./~/react-router-dom/es/StaticRouter.js 55 bytes {0} [built]
 [516] ./~/react-router-dom/es/Switch.js 49 bytes {0} [built]
 [517] ./~/react-router-dom/es/matchPath.js 52 bytes {0} [built]
 [542] multi ./src/index.js 28 bytes {0} [built]
    + 528 hidden modules
^C%                                                                                                                                                                 user@host /tmp/webengineering-2017/src/main/frontend: ➦ a3ac089: 
```

Also, the heroku build failed after merging #20. With this PR it's fine.

mischah commented:
Ahh … now I see 🙈

`webpack.config.js` wasn’t changed in #20. 

Thanks 👌 

So @mlesniak better merge that in 😘
